gpkit/_mosek/expopt.py

```diff
@@ -66,14 +66,14 @@ def __getattr__(self, attribute):
                             "DUAL_FEAS",  # switched with below for GP
                             "PRIM_FEAS",
                             "PRIM_AND_DUAL_FEAS",
-                            "PRIM_INFEAS_CER",
                             "DUAL_INFEAS_CER",  # switched with below
+                            "PRIM_INFEAS_CER",
                             "NEAR_OPTIMAL",
                             "NEAR_DUAL_FEAS",  # switched with below
                             "NEAR_PRIM_FEAS",
                             "NEAR_PRIM_AND_DUAL_FEAS",
-                            "NEAR_PRIM_INFEAS_CER",
                             "NEAR_DUAL_INFEAS_CER",  # switched with below
+                            "NEAR_PRIM_INFEAS_CER",
                             "INTEGER_OPTIMAL",
                             "NEAR_INTEGER_OPTIMAL"]
```

Comment:
The "switched with below" comments are now confusing -- did these changes undo those switches? Does this list match the mosek documentation (or some source somewhere)? Or is it our own creation?